### DISCUSSION TITLE
Welcome to PMGH Discussions

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#396] [(A) Anything]
## Welcome to PMGH Discussions!

Here, you can suggest new hacks, discuss bugs, find commonly asked questions, or just chat with us :) 

## Common Links

* [Official Discord](https://discord.gg/XQDfbfq);
* [Installation Instructions](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/wiki/How-to-install-hacks)

**Comment by ArcerionDev** [comment]
### Discussion rules:
- Please be kind to everyone in the chat
- No repeat questions
- No spamming
More rules will be added if needed.

**Reply by ghost** [reply]
Can you find a way for me to join back into y'all discord server? They banned because I was 13 years old and I need to be updated if like the hack is down

**Reply by ghost** [reply]
Is there a hack that can save my level?

**Reply by PatheticMustan** [reply]
1. I can unban you. What's your username?
2. Level hacks should automatically save

**Reply by ghost** [reply]
My username is Alex Chen#8234

**Reply by ghost** [reply]
Yeah I know it only makes my level 100 ai want to find other ways to make my level and save it

**Reply by ghost** [reply]
Can you tell me when I can join y’all server again?

**Reply by PatheticMustan** [reply]
you're not banned, join with discord.gg/XQDfbfq

**Reply by ghost** [reply]
It says the link is invalid or expired

**Reply by ghost** [reply]
You have ChiefNotch007 discord server? If you do can you send me a invite

**Reply by PatheticMustan** [reply]
https://discord.gg/a5Z84qfm

**Reply by ghost** [reply]
I asked Chief to send me a invite link and it said I was banned

**Reply by ghost** [reply]
The link you sent me did not work either

**Reply by ghost** [reply]
Maybe it is my username try Poop#8234

**Reply by PatheticMustan** [reply]
not banned

**Reply by ghost** [reply]
PMGH did ban me because I was there when they did it and it keeps on saying unable to accept invite

**Reply by ghost** [reply]
Why does it freeze when I give people hacked Prodigy accounts

**Reply by ghost** [reply]
Can you send the discord link?

**Reply by ghost** [reply]
Your arena point hack did not work

**Comment by yokisniff** [comment]
Ok. What is this "Harmony Island" place and why is it locked for me??
P.S: You guys should update the map or the WCM because that 'harmony island' just came out
<img width="960" alt="er" src="https://user-images.githubusercontent.com/75645436/101825839-3167fc00-3afc-11eb-87dd-d57287c2c0cf.png">

**Reply by ghost** [reply]
I’m not sure so far I’m exploring it right now.

**Reply by ghost** [reply]
So far it’s very buggy for me.
![922799A5-4ED7-4F6E-A85B-E290B2D1BC4B](https://user-images.githubusercontent.com/74740438/101829344-ba286d00-3ae7-11eb-8c49-5595d41baf69.jpeg)

**Reply by ghost** [reply]
Ooh I realized I can’t use pets that are unavailable in the game I had crystal monster.

**Reply by yokisniff** [reply]
**oh**

**Reply by ghost** [reply]
That’s exactly what happened to me.

**Reply by ghost** [reply]
https://prodigy-math-game.fandom.com/wiki/Harmony%20Island

**Comment by PatheticMustan** [comment]
wow, it's a new island! That's pretty neat

**Reply by ghost** [reply]
So far no luck on doing the ruin rune. Too much players were online earlier.
Edit 8:16 PM PST: Still can’t do it. Does anyone know why?

**Reply by yokisniff** [reply]
server for that patch-whatchamicallit is full so you can't

**Reply by josiah1393** [reply]
you have to be on level 20 and have any hacks about the epics I am on level 70

**Reply by yokisniff** [reply]
i am level infinity
![image](https://user-images.githubusercontent.com/75645436/102153817-0821d580-3e46-11eb-9e24-452f0047e6bb.png)

**Reply by ghost** [reply]
Did the level save if it did tell me how to do it. I can make my level but I have to use the bookmarklet maker

**Reply by afkvido** [reply]
@DerpKiddo HOW did u get infinity?

**Reply by donaldli2020** [reply]
> @DerpKiddo HOW did u get infinity?

```
_.player.data.level = Infinity;
```

**Reply by afkvido** [reply]
i was too dum to notice…

**Reply by yokisniff** [reply]
> @DerpKiddo HOW did u get infinity?

lol

**Comment by yokisniff** [comment]
<h1>Why is the map hack not working?</h1>

When I battle someone and if they're slow, I usually use that hack to teleport to somewhere else. But now it just does nothing.

**Reply by PatheticMustan** [reply]
Sorry! We'll look into this as soon as possible

**Reply by yokisniff** [reply]
ok

**Reply by josiah1393** [reply]
how do i get on level infity or is it copy bait thats what my sister did

**Reply by aj5151** [reply]
how do you hack

**Reply by aj5151** [reply]
im new

**Reply by yokisniff** [reply]
> how do i get on level infity or is it copy bait thats what my sister did
@josiah1393

I used tampermonkey and coded in javascript and then went to inspect in the console

**Reply by yokisniff** [reply]
> how do you hack
@aj5151 it's not easy some are illegal

**Reply by ghost** [reply]
yeah but ilellegal fun

**Reply by gotoprodighy** [reply]
Can someone hack my prodigy acc

**Reply by ghost** [reply]
So how do I use tampermonkey?

**Comment by yokisniff** [comment]
**<h6>When did you guys make these hacks they are awesome</h6>**

**Reply by PatheticMustan** [reply]
We started around September 2019... :D

**Reply by yokisniff** [reply]
Ooh these hacks are awesome

**Reply by ghost** [reply]
I am actually one of the OG’s. My friend told me about it around September last year. I didn’t have a GitHub account back then so I don’t have much proof.

**Reply by yokisniff** [reply]
> I am actually one of the OG’s. My friend told me about it around September last year. I didn’t have a GitHub account back then so I don’t have much proof.

@XxDankGod96xX
oh

**Comment by PatheticMustan** [comment]
We've had reports that morphing into the crystal monster breaks the game. If morphing crashes your account, use "Fix Morph Crash"

**Reply by yokisniff** [reply]
got it!

**Reply by ghost** [reply]
> We've had reports that morphing into the crystal monster breaks the game. If morphing crashes your account, use "Fix Morph Crash"

The "Fix Morph Crash does not work for me."

**Reply by ghost** [reply]
Run both "Fix Morph Crash" and "Fix Battle Crash", then refresh the page and login again.

**Comment by josiah1393** [comment]
do you have any hack to get chil and char those are my fave

**Reply by ghost** [reply]
bruh use the get pets yesbutton

**Comment by josiah1393** [comment]
I do a little hacking but some reason they don't work no more thanks to
harmony island

**Comment by PatheticMustan** [comment]
Can somebody make the installation instructions more obvious on the README, idk make it super big so skids ABsolutely can't miss it, getting tired

**Reply by ghost** [reply]
yes. i keep telling people to go to this repo and they're like "ALL I SEE IS A BUNCH OF CODE AND FILES WHERE IS THIS README" and its so dumb

**Comment by josiah1393** [comment]
hey i did not see it😥

**Comment by josiah1393** [comment]
hey in the arena if you open another tab and wait a couple min. then you
win but you must pick a spell so they think you try then you win

**Reply by PatheticMustan** [reply]
that's not hacking, that's just being incredibly rude to whoever you're battling

**Reply by josiah1393** [reply]
not rude  does hacking mean cheating and making yourself the best hmm 🤔🤔🤔 not rude

**Reply by PatheticMustan** [reply]
you're literally afk'ing in a match with a real person, wasting their time, I don't understand what's not rude about it

**Reply by josiah1393** [reply]
HMM SOMEBODYS MAD BUT HEY I GOT ALL THE PRIZES AND ON TOP THOUSEND

**Reply by josiah1393** [reply]
thats ALL i wanted to do @PatheticMustan

**Reply by PatheticMustan** [reply]
... you know you can do "Get All xxx" and get literally everything possible in the game, right?

**Reply by josiah1393** [reply]
how

**Reply by josiah1393** [reply]
oijgiyftrdcfvghhugyftdtychjfxc8trxc8trcd96rtfg97tf

**Comment by josiah1393** [comment]
😥😥😥

**Comment by josiah1393** [comment]
does hacking mean cheating and making yourself the best hmm 🤔🤔🤔 not rude

**Comment by yokisniff** [comment]
I have a question. So there is a thing where you can edit your pet's spells, right? In the arena, can players see it?

**Reply by PatheticMustan** [reply]
yes.

**Reply by yokisniff** [reply]
oh nice

**Comment by yokisniff** [comment]
There is a discussion, but on the readme it says error 
![image](https://user-images.githubusercontent.com/75645436/102381924-d28d0180-3f97-11eb-9ae2-b0aa0b0916e2.png)

**Reply by josiah1393** [reply]
yeah

**Reply by yokisniff** [reply]
confused

**Reply by josiah1393** [reply]
yeah

**Reply by PatheticMustan** [reply]
thanks, fixed

**Reply by yokisniff** [reply]
> thanks, fixed

@PatheticMustan yay

**Comment by poIarbear** [comment]
it's in the github but prodigy will reset you back to level 100 if you
refresh

**Reply by josiah1393** [reply]
what

**Reply by PatheticMustan** [reply]
levels are capped at 100.... anything beyond breaks multiplayer battles and other things, so we usually don't recommend having levels past 100.

**Comment by josiah1393** [comment]
does anyone know this person in prodigy abigail airspoon that person is a hacker i batteld her they are on level 4 an she killed me and my pets and that was no all out attack that was a hack

**Comment by yokisniff** [comment]
When I do this <a href="https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/hacks/scriptBasedHacks/ArenaPoints.js">hack</a>, it won't give me points when I beat someone. Pls fix this

**Reply by josiah1393** [reply]
wait what english please what do   that even mean

**Reply by PatheticMustan** [reply]
arenapoints hacks work by spamming the win endpoint. This endpoint is ratelimited to 1/minute. If you win, it'll go beyond the rate limit. Just.... don't battle??

**Reply by josiah1393** [reply]
WAIT IS THAT TO ME

**Reply by josiah1393** [reply]
![imgpsh_fullsize_anim](https://user-images.githubusercontent.com/75404852/102671853-52ac9600-4155-11eb-8241-6c73d4fb2887.gif)

**Reply by PatheticMustan** [reply]
> WAIT IS THAT TO ME

no, i wasn't talking to you

**Reply by yokisniff** [reply]
> arenapoints hacks work by spamming the win endpoint. This endpoint is ratelimited to 1/minute. If you win, it'll go beyond the rate limit. Just.... don't battle??

@PatheticMustan  okay?? ok

**Comment by ghost** [comment]
I accidentally figured out how to use Arc’s Cheat Menu and a few other hacks on IOS...
I tried using the Shortcuts App to see if I can use the Run JavaScript On Webpage command, and it would always say some kind of error, like “Can’t Find Variable “_”   And I was messing around trying to make it so I can just copy what script I want and run the shortcut. (Basically, you can insert built in “variables” inside the text not the “var hello” they’re not supposed to be used in the script, but if you use variables and set the variables as the script you want, it will still say errors but run the script anyways. (I’ll send a photo from my laptop soon.)

**Reply by ghost** [reply]
![](https://www.linkpicture.com/q/8B1F9381-4271-4ED5-9A19-8DC4032CB229.png)

![](https://www.linkpicture.com/q/E1DA06A1-1FC6-4DFB-B167-6899CF01F1AC.jpeg)

**Reply by ArcerionDev** [reply]
the reason that you can't find `_` is because it is created when you install the chrome extension. otherwise, it's not accessible.

**Reply by ghost** [reply]
what if we (you) made a network-wide "vpn" that like isnt a vpn but you put it on your router to add cheat menu on all device

**Reply by ArcerionDev** [reply]
that's actually an interesting idea

**Comment by ghost** [comment]
Note: The only thing that works on Arc’s Cheat menu is the “ArenaPoint Generator” only because it does not require a reload. If you need the script for Arcs Cheat menu tell me, And I’ll paste it here

**Reply by ghost** [reply]
Also it works with the codeEvaluator

**Reply by PatheticMustan** [reply]
for future reference, you can directly run javascript inside of a bookmark.

`javascript:` + whatever code you want

if you wanted to run `alert(1)`, it would be

```
javascript:alert(1)
```

**Reply by ghost** [reply]
Oh, Thanks.

**Reply by josiah1393** [reply]
best hacker now i cracked prodigy i will not tell you how

![7e2b6c42df1d15add6de0ccf46b15de7](https://user-images.githubusercontent.com/75404852/104060360-64191900-51bc-11eb-927f-af4baa8c2653.jpg)

**Comment by poIarbear** [comment]
lolll

**Comment by aj5151** [comment]
how do you hack

**Reply by ghost** [reply]
magic

**Comment by doormatt-cool** [comment]
owo turns the word come to "cum"

**Reply by ghost** [reply]
0-0

**Reply by josiah1393** [reply]
owo

**Comment by aj5151** [comment]
what do you mean by "cum"
On Mon, Jan 4, 2021 at 12:06 PM • ͜ • <<notifications@github.com>notifications@github.com> wrote:
0-0

**Reply by ArcerionDev** [reply]
cum

**Reply by ghost** [reply]
cum

**Reply by ghost** [reply]
cum = semen = comes out of human dick  whn hacvig sx

**Comment by josiah1393** [comment]
i think if you guys are helping prodigy y hack it yall must be dumb and yall hack a kid math game huh slow peolple and hey who care im 11

**Reply by ghost** [reply]
> i think if you guys are helping prodigy y hack it yall must be dumb and yall hack a kid math game huh slow peolple and hey who care im 11
when you hire people to hack your company, it lets you know the vulnerabilities of your website ; which helps you raise your defenses.
I don't know why your saying this like you know everything about hacking, but it's not that simple. This kind of hacking is a mix between "Ethical Hacking" and Cheating. But mostly Cheating.
Point is companies will hire hackers, and will pay handsomely.
Hacking will help a lot.

**Reply by josiah1393** [reply]
so i hate this hacking grup alll of you 
@PatheticMustan @Lenny-Face @ArcerionDev

**Reply by josiah1393** [reply]
that means you hack because you c ant win
silly

**Reply by ghost** [reply]
ya i cant win

**Reply by PatheticMustan** [reply]
> that means you hack because you c ant win
> silly

https://github.com/Prodigy-Hacking/ProdigyMathGameHacking#motivation

**Comment by josiah1393** [comment]
yall all must be dumb y hack a kids game and yall must be slow and im 11 who cares

**Reply by josiah1393** [reply]
silly
fake hackers

**Reply by ArcerionDev** [reply]
> yall all must be dumb y hack a kids game and yall must be slow and im 11 who cares

you have to be 13+ to use github

**Reply by PatheticMustan** [reply]
Ban....? 👀

**Reply by ArcerionDev** [reply]
^^

**Reply by josiah13934** [reply]
back again yall still suck

**Reply by josiah13934** [reply]
ban me again i will still get in yall retarted selfs

**Reply by josiah13934** [reply]
dang yall hack a kids game so silly
@PatheticMustan @ArcerionDev @Lenny-Face

**Reply by josiah13934** [reply]
ban u

**Reply by josiah13934** [reply]
@PatheticMustan  this one dont like @KryptoCrash

**Reply by KryptoCrash** [reply]
> @PatheticMustan this one dont like @KryptoCrash

Hello there, thanks for pinging me. I’d like to remind you by using the hacks you essentially gave us your ip, which is the equivalent of your online social security number. There is also enough information about you posted publicly by you that I can trace down your ip from previous Redirector requests. I’d advise you to stop bothering the admins and go annoy some other repo instead.

**Reply by ghost** [reply]
what penguin man said

**Reply by ArcerionDev** [reply]
lmaooo

**Reply by ghost** [reply]
also when you view the readme that gets your IP too and your wifi name and password and address and all the info on your computer and it's all saved on pmgh cloud servers

**Reply by josiah13934** [reply]
also i have never readme oh also my comuter  cant get glitched all private

**Reply by josiah13934** [reply]
oh also  never used these hacks

**Reply by KryptoCrash** [reply]
Bruh don’t publicly dox that’s cringe.

**Reply by ArcerionDev** [reply]
and why are you here if you haven't used the hacks

**Reply by ArcerionDev** [reply]
> Bruh don’t publicly dox that’s cringe.

sorrrryyy

**Reply by ghost** [reply]
> also i have never readme oh also my comuter cant get glitched all private

actually when you visit the home page of this repository it shows the README at the bottom. and when you comment here it does the same.

**Reply by ghost** [reply]
> Ban....? 👀

Time for an IP ban 😈😈😈 (jkjk XD)

**Comment by ghost** [comment]


**Reply by yokisniff** [reply]
what the heck is wrong with you

**Reply by ghost** [reply]
> what the heck is wrong with you

everything

**Comment by Arcerion-Dev** [comment]
Would it be possible if you can update Royal Roar in willsCheatmenu Edit Pet’s Spells?
It’s still the old royal roar, and that one doesn’t exist anymore so when you use the spell, it either crashes right when the battle starts or
does nothing. By that I mean when you finish the math problem it just goes back to the battle screen and nothing happens it just waits eternally.
So, is there any way you can fix this?

**Reply by ArcerionDev** [reply]
open an issue if you'd like

**Reply by Arcerion-Dev** [reply]
Ok, Thanks.

**Comment by yokisniff** [comment]
what is the arena point generator

**Reply by ArcerionDev** [reply]
currently not online

**Reply by yokisniff** [reply]
> currently not online

@ArcerionDev  ok sad but ok

**Comment by ArcerionDev** [comment]


**Reply by yokisniff** [reply]
?

**Comment by jaheima** [comment]
can you help me get infnite counjering cubes i cant figure it out and if so i will tell you my account info

**Reply by PatheticMustan** [reply]
please don't. There should be a button on the cheat menu.

**Comment by ghost** [comment]
can i make a thing where i sell hacked accounts to mobile users as long as i credit this repository

ty

~ a script kiddie

**Reply by ghost** [reply]
wait selling is against your conduct of code

okay so i'll give them away for free but i'll insert just a few ads into my shop (not behind a paywall and im not putting ads in the hacks) and accept donations

good???

**Reply by ghost** [reply]
https://github.com/NameIsA/MobileDevice-ProdigyHacking/blob/main/README.md

**Reply by PatheticMustan** [reply]
minor ads are fine, but pls no video ads

as long as you don't charge for one account, and link to the original repo, I think it should be fine.

**Reply by ghost** [reply]
okay

**Comment by ghost** [comment]
Hi, I've created a station for you to request a hacked account if you use a mobile device. I do not hack existing accounts, because, in the event that my email was compromised, any account logins sent to me would be compromised as well, so I do not want to risk your security- I am sorry for any incovience this causes.

Use [this form](https://nameisa.github.io/MobileDevice-ProdigyHacking/) to request a hacked account. You can have one account for free, but I really don't want to go out of my way to make excessive alts for people for no reason, so if you want a bunch of accounts, that costs a fee. However, if your account is broken, I will attempt to repair it for free. If I am unable to repair it, you will get a new account for free.

This is not an official service of PMGH, it's made by myself - a community member - as PMGH does not support giving out/logging into accounts for others [for a variety of legitimate security reasons](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions/170), which is why I suggest you to consider your choice before requesting me to create an account. Specifically,
* If my email was compromised, so are all the accounts shared in it.
* It is up to you to trust if you want to trust me with the login details to your new hacked account.

Note that orders may take a bit, I have been quite busy recently.

~NameIsA

(Copied from [this post, that I made](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions/487).)

**Comment by elikiller** [comment]
how do i get the reviewer bot on my discord server can someone help me plz?

**Reply by elikiller** [reply]
wait nvm that

**Comment by elikiller** [comment]
Does anyone know how I can get the shortcuts app without the App Store? cos I’m on iPad so I can’t use the hacks without the shortcuts app

**Reply by PatheticMustan** [reply]
first, you can't use our hacks on mobile devices, since you can't install the extensions.
second, you can run javascript by making a bookmark, and prepending "javascript:" to the code.

As an example, `alert(1)` would be changed to `javascript:alert(1)`

**Reply by elikiller** [reply]
ok thanks! :)

**Reply by ghost** [reply]
https://nameisa.github.io/MobileDevice-ProdigyHacking/ if you want

**Reply by ArcerionDev** [reply]
istg dck I will block you for advertising, ***please stop***

**Reply by ghost** [reply]
im not advertising, im just linking to another project... thats not bad!

**Reply by elikiller** [reply]
> first, you can't use our hacks on mobile devices, since you can't install the extensions.
> second, you can run javascript by making a bookmark, and prepending "javascript:" to the code.
> 
> As an example, `alert(1)` would be changed to `javascript:alert(1)`

What browser do I do this in for the hacks to work?
Safari, or Chrome?

**Reply by ghost** [reply]
> https://nameisa.github.io/MobileDevice-ProdigyHacking/ if you want

What’s the point of it if you just release the shortcut to the public, like PMGH extension.

**Reply by gotoprodighy** [reply]
Can someone hack my prodigy acc pls

**Reply by ghost** [reply]
> Can someone hack my prodigy acc pls

No. It violates your security, but I will be doing a hacked account giveaway soon on my YT channel if you're interested.

**Comment by Vannakreach** [comment]
omg


On Sat, Feb 6, 2021 at 5:17 AM savage8685 <notifications@github.com> wrote:

> Can someone hack my prodigy acc pls
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions/396#discussioncomment-343186>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQ7R7NGTIIKXIDGBDXS2FR3S5RU7RANCNFSM4USZMQMQ>
> .
>


-- 
Vannakreach

**Comment by elikiller** [comment]
Awww why doesn’t bookmarklets work? :(((((

**Reply by PatheticMustan** [reply]
no longer supported

**Reply by gotoprodighy** [reply]
Can someone hack my acc or give me a free acc for prodigy

**Reply by ghost** [reply]
I use bookmarklets to make my lvl infinity, make my damage infinity, do infinity damage, and 
make my hp max hp,

**Reply by ghost** [reply]
Why don't you just hack Prodigy?

**Reply by elikiller** [reply]
How do u do this I’ve been trying for ages @achen6196

**Comment by gotoprodighy** [comment]
Does anybody a free acc for me pls

**Reply by ghost** [reply]
Make it yourslef

**Reply by ghost** [reply]
If you're a mobile user (or are otherwise unable to use the hacks), ask @CRobbins0867. If you can install them yourself, just use Will's Cheat Menu.

**Comment by ghost** [comment]
The link did not work because I am banned I asked Chief to send me a invite
link and it said I was banned

**Comment by ghost** [comment]
So do you know a hack that will save my level like if I made my level 60000
is their a hack that will save that level

**Comment by NessiePUYOY** [comment]
Can you please make it possible to save levels on the game plss also can you add a thing on the cheat menu to get hacked hairstyles that no other people can get because i have seen people with that type of stuff plsss it would help alot
I love you guys,

Nessie /<3/

**Reply by PatheticMustan** [reply]
I'll think about the hairstyles, but the levels are already saved

**Reply by PixelMMO** [reply]
the levels dont save

**Reply by ghost** [reply]
If you make the lvl like 1e+90 it will just be lvl 100 again but I use bookmarklet so it don't really matter if it does not save for me because I can just log into Prodigy and click on the bookmarklet and it will make my lvl infinity

**Comment by PixelMMO** [comment]
how do i get to harmony island now? harmony island isnt on the interactive map

**Reply by ArcerionDev** [reply]
it's at the very bottom I believe

**Reply by ArcerionDev** [reply]
![image](https://user-images.githubusercontent.com/60945964/109526915-1b296680-7a81-11eb-8946-b32a72d43b04.png)

**Reply by PixelMMO** [reply]
ty

**Comment by PixelMMO** [comment]
i suggested lots of things in the suggestions discussion and im waiting for someone o reply, i was hoping you could reply

**Comment by PixelMMO** [comment]
@PatheticMustan @ArcerionDev 
![image](https://user-images.githubusercontent.com/79719521/109704177-4df85b00-7b64-11eb-9141-b975f86bb524.png)
is that a problem?

**Reply by ghost** [reply]
Do you have HTTPS Everywhere or use a school device?

**Reply by ghost** [reply]
That happens to me too

**Reply by ghost** [reply]
What is HTTPS?

**Reply by ghost** [reply]
Hypertext Transport Protocol Secured

HTTP = How websites and the internet talk to each other

HTTPS = Encrypting how websites and the internet talk to each other

Encryption is a double-edged sword. It prevents accounts/services from being compromised, but makes modifications harder.

**Comment by ghost** [comment]
Hey so y’all banned me from y’all server because I was 12 and I just turned 13 today and I deleted my old account that I got banned from and is there a way y’all can unban me?

**Reply by ghost** [reply]
I think that why y’all can’t unban me because I deleted my old discord account and y’all have to unban me with my old discord account but it deleted

**Reply by ghost** [reply]
@PatheticMustan @ArcerionDev

**Reply by ghost** [reply]
Why did you call for them?

**Reply by ghost** [reply]
they're the most active discord admins

**Reply by ghost** [reply]
Oh do you think they can find a way to unban me even though they banned me with my account that I got banned from?

**Reply by ghost** [reply]
> Oh do you think they can find a way to unban me even though they banned me with my account that I got banned from?

yes

they click the server then click the name then click manage server then click bans then click your username then click unban

takes about 15 seconds

**Reply by ghost** [reply]
So you hack Prodigy?

**Reply by ghost** [reply]
I do it fun

**Reply by ghost** [reply]
But I got banned from my deleted account

**Reply by ghost** [reply]
> So you hack Prodigy?

i code hacks because prodigy is being ignorant to the issues at hand and im going to get their attention by demonstrating how dangerous it is to let a bunch of teenagers let a bunch of kids click one button to win their "educational" game

**Reply by ghost** [reply]
I don’t know if they can unban me

**Reply by ghost** [reply]
> I don’t know if they can unban me

bruh

> yes
>
> they click the server then click the name then click manage server then click bans then click your username then click unban
>
> takes about 15 seconds

**Reply by ghost** [reply]
Oh okay

**Reply by ghost** [reply]
So they click deleted user to unban me?

**Reply by ghost** [reply]
erm

if your old account is deleted, wait 14 (or was it 30?) days then it's IP is removed from discord

then you can just join back and the ban wont stick

**Reply by ghost** [reply]
So they unban my deleted account and I wait 14 days to join back?

**Reply by ghost** [reply]
@PatheticMustan can you unban my old account?

**Reply by ghost** [reply]
Can you tell me once you unbaned my deleted account?

**Reply by ghost** [reply]
> So they unban my deleted account and I wait 14 days to join back?

if they just unban your deleted account it will go away instantly but uhh idk how they tell your deleted account apart from the others

**Reply by ghost** [reply]
So how do they unban people?

**Reply by ghost** [reply]
> So how do they unban people?

very simple

> they click the server then click the name then click manage server then click bans then click your username then click unban
>
> takes about 15 seconds

i dont know how you miss that maybe you're just an old boomer with dementia or something but i've said it 2 times

**Reply by PatheticMustan** [reply]
ah yes, I'll just unban... the deleted... account....
![image](https://user-images.githubusercontent.com/35382971/109827401-af263a00-7c09-11eb-837c-e0a45f413329.png)

**Reply by ghost** [reply]
thats kinda what i was thinking @PatheticMustan

**Reply by ghost** [reply]
That a lot of deleted accounts

**Reply by ghost** [reply]
So your gonna unban all of them?

**Reply by ghost** [reply]
bruh

**Reply by PatheticMustan** [reply]
no

**Reply by ghost** [reply]
So how you gonna unban me?

**Reply by ghost** [reply]
we dont know and we also dont know that you turned 13 and like you dont act like it lmao

**Reply by ghost** [reply]
I did

**Reply by ghost** [reply]
Just saying but why do you complain so much?

**Reply by ghost** [reply]
> Just saying but why do you complain so much?

bruh

**Reply by ghost** [reply]
You said you were gonna the deleted accounts

**Reply by ghost** [reply]
what  were the 4 numbers that were after your username on discord

**Reply by ghost** [reply]
Um I forgot

**Reply by ghost** [reply]
bruh

**Reply by ghost** [reply]
what was your username

**Reply by ghost** [reply]
Poop

**Reply by ghost** [reply]
ah i remember you

**Reply by ghost** [reply]
You know my number?

**Reply by ghost** [reply]
no because i got banned for debating copyright in the pmgh server

**Reply by abish1443** [reply]
Can someone make me a hacked prodigy account i am on a school device and have no way to use the hacks

**Reply by ghost** [reply]
ask @CRobbins0867

**Comment by achen6067** [comment]
Can y'all unban me?

**Reply by achen6067** [reply]
@PatheticMustan are you gonna unban me?

**Reply by PatheticMustan** [reply]
shut up, I already unbanned you three times

**Reply by NX721** [reply]
lmao

**Comment by NessiePUYOY** [comment]
Hey I  was thinking of what type of hacks we can add to the cheat menu and i think it would be really cool if you could change the color of our clothes on the game like for example, change the spector robes a purple pink blue etc and also the morph thing is a  big problem still it keeps freezing up, and the new thing you added where we can change our name to whatever we wanted, nobody can see it but me so whats the fun in that please message back to fix these two problems and possibly add a hack to change our clothing colors :)

**Comment by ghost** [comment]
(sadly) isnt possible

prodigy is making the new art so hideous because they're going to make this
a feature apparently

>

**Reply by elikiller** [reply]
what script do I put into console for wcm

**Reply by ghost** [reply]
https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/wiki/How-to-install-hacks

**Reply by ghost** [reply]
it can take up to 20 seconds to appear

**Reply by elikiller** [reply]
I tried it stayed on halfway loading :(

**Reply by ghost** [reply]
oh something's wrong

@PatheticMustan @ArcerionDev @BoredFishRE

**Reply by ghost** [reply]
submit a new issue

**Reply by elikiller** [reply]
but i used the bookmark that clears the console so it wouldn't but it still did it here's the code: javascript:console.log=console.clear=console.dir=_=>{}

**Reply by elikiller** [reply]
ok

**Reply by elikiller** [reply]
@ChickenOnYT my admin blocked the extension in the web store :/

**Reply by elikiller** [reply]
RIGHT fuking after they released the newer version onto it :////////

**Reply by ghost** [reply]
> @ChickenOnYT my admin blocked the extension in the web store :/

ah that sucks

**Reply by elikiller** [reply]
I found a way to get phex but it still says that typeerror thingy that u said means they're blocked

**Reply by BoredFishRE** [reply]
> 
> 
> I found a way to get phex but it still says that typeerror thingy that u said means they're blocked

do you have PHEx 2.0.3?

**Reply by elikiller** [reply]
yea tried it loaded halfway and it had the type error @BoredFishRE

**Reply by NX721** [reply]
> yea tried it loaded halfway and it had the type error @BoredFishRE

Well, restarting your device will help.

**Comment by achen6067** [comment]
Can y'all unban me?

**Reply by ghost** [reply]
@ArcerionDev

**Comment by irhrhd** [comment]
On discord if you got banned it was for a reason

**Reply by achen6067** [reply]
It was for a dumb reason

**Reply by achen6067** [reply]
All I did was show something funny

**Reply by ghost** [reply]
ah you got muted for spamming gifs i remember correctly

**Reply by achen6067** [reply]
Hey wanna see something funny @ChickenOnYT?

**Reply by achen6067** [reply]
Um can anyone unban me?

**Reply by BoredFishRE** [reply]
No. Stop asking. If you don't stop, I will ban you from here

**Reply by NX721** [reply]
Rip

**Reply by achen6067** [reply]
Can you unban me?

**Reply by BoredFishRE** [reply]
Alright that's it, I warned you

**Reply by achen6067** [reply]
Hi @BoredFishRE

**Reply by BoredFishRE** [reply]
We're not unbanning you, and stop asking

**Reply by NX721** [reply]
lmao

**Reply by ghost** [reply]
> Hey wanna see something funny @ChickenOnYT?

it would be really funny if you jumped off a cliff so can you do that bitch

**Reply by BoredFishRE** [reply]
We banned him from the discord and the github now.

**Reply by BoredFishRE** [reply]
So he ain't gonna reply

**Reply by ghost** [reply]
ah i see

**Reply by NX721** [reply]
> So he ain't gonna reply

He'll make more acc

**Comment by MATEMATIK-SOSYE** [comment]
The "Edit Pet" option has been removed after upgrade. Why is that so? 🤔

**Reply by ghost** [reply]
it was breaking stuff

**Reply by MATEMATIK-SOSYE** [reply]
> it was breaking stuff

Well, it was _still_ wourking just fine for me. My account still wourks to the this day. Why not put the button back up there? 😉

**Reply by MATEMATIK-SOSYE** [reply]
Also, after answering my question, I have something very eerie to report that happened with my CURRENT selected pets after I got offline from the arena _approximately_ **three days** ago. Hopefully you can help me out.

**Reply by NX721** [reply]
Screenshot. -_-

**Reply by MATEMATIK-SOSYE** [reply]
> Screenshot. -_-

Pardon?

**Reply by NX721** [reply]
Screenshot the _offline Arena_

**Reply by NX721** [reply]
> > it was breaking stuff
> 
> Well, it was _still_ wourking just fine for me. My account still wourks to the this day. Why not put the button back up there? 😉

The code is still there just not active or offline.

**Comment by achen2341** [comment]
Why do y'all keep deleting my issue?

**Reply by ghost** [reply]
idk maybe because you've ban evaded 9 times and spammed and we arent gonna unban you

**Reply by NX721** [reply]
Lmao 9 times Alex Chen: Almost at 15 ban evasions

**Reply by ghost** [reply]
this guy is alex chen he's at 15 now holy shit

**Reply by NX721** [reply]
Oops guess I haven't been counting Lmao
I think he'll beat your record Lol

**Reply by NX721** [reply]
> this guy is alex chen he's at 15 now holy shit

Bruh @achen2341  You'll beat @ChickenOnYT 's record, Holy wtf

**Reply by NessiePUYOY** [reply]
Hey I have a new idea for PMGH I think it would be cool if you made new faces to put on our character and dances I would be soooooooooooo very happy if its possible <3

**Reply by ghost** [reply]
good idea

**Comment by ghost** [comment]
@ChickenOnYT  PLS DO NOT SAY BAD WORDS IN CHAT!!!!!
